### PR TITLE
Dawnpm system banner

### PR DESCRIPTION
**Pull request by dawnpm**

### CODE REVIEW

File: _includes/navbar.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
         </div>
       </form>
       <div class="nav navbar-right">
-          <a href="https://search.usa.gov/sites" class="navbar-brand"><i
+          <a href="https://search.gov/system-banner.html" class="navbar-brand"><i
```
Comment (MothOnMars) [inline _includes/navbar.html]: @dawnpm , I was going to suggest making these all relative links:
```html
          <a href="https://search.gov/system-banner.html" class="navbar-brand"><i
              class="icon-user icon-white"></i>&nbsp;Login</a>
...
          <a href="http://search.usa.gov/signup" class="navbar-brand">Sign up</a>
...
          <a href="https://search.gov/status.html" class="navbar-brand">System Status</a>
```
However, I'm seeing plenty of SEO articles that recommend absolute links, so let's keep 'em that way. (But let's update `http://search.usa.gov/signup` to `https`.)

File: _posts/2019-10-08-system-banner.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+permalink: /system-banner.html
+layout: post
+title: Search.gov Login Terms
+tags: status
+description: This page provides access to the Search.gov Application.
```
Comment (MothOnMars) [inline _posts/2019-10-08-system-banner.md]: @dawnpm , do we want this page to be indexed & searchable? If not, let's add these tags:
```
sitemap: false
metatags: noindex
```

File: _includes/navbar.html (hunk 1 above)
Comment (dawnpm) [reply _includes/navbar.html]: good catch on the http:, thanks.

File: _posts/2019-10-08-system-banner.md (hunk 2 above)
Comment (dawnpm) [reply _posts/2019-10-08-system-banner.md]: Good call, putting those in.

**Comment by MothOnMars** [comment]
LGTM 👍

**Pull request closed without merging**